### PR TITLE
libresplit: init at 0-unstable-2024-06-05

### PR DESCRIPTION
**Pull request by fgaz**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/li/libresplit/package.nix [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  buildInputs = [
+    gtk3
+    jansson
+    luajit
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  passthru.updateScript = unstableGitUpdater {
+    url = "https://github.com/wins1ey/LibreSplit.git";
```
Comment (AndersonTorres) [inline pkgs/by-name/li/libresplit/package.nix:40]: Redundant

Comment (fgaz) [reply pkgs/by-name/li/libresplit/package.nix:40]: https://github.com/NixOS/nixpkgs/blob/b6050e73492a4d129b5fd3cb757804a918a9665e/pkgs/common-updater/unstable-updater.nix#L84

Nice!

Should I just remove the url or is the whole updater redundant?

Comment (fgaz) [reply pkgs/by-name/li/libresplit/package.nix]: Apparently just the url is redundant (I tried both update.nix and nix-update)

**Review by AndersonTorres — APPROVED** [review]
Result of `nixpkgs-review pr 341396` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>libresplit</li>
  </ul>
</details>

Warning: empty dir found:  share/glib-2.0

```shell
$> tree results/libresplit/
results/libresplit/
├── bin
│   └── libresplit
└── share
    ├── applications
    │   └── libresplit.desktop
    ├── glib-2.0
    ├── gsettings-schemas
    │   └── libresplit-0-unstable-2024-06-05
    │       └── glib-2.0
    │           └── schemas
    │               ├── gschemas.compiled
    │               └── libresplit.gschema.xml
    └── icons
        └── hicolor
            ├── 128x128
            │   └── apps
            │       └── libresplit.png
            ├── 16x16
            │   └── apps
            │       └── libresplit.png
            ├── 22x22
            │   └── apps
            │       └── libresplit.png
            ├── 24x24
            │   └── apps
            │       └── libresplit.png
            ├── 256x256
            │   └── apps
            │       └── libresplit.png
            ├── 32x32
            │   └── apps
            │       └── libresplit.png
            ├── 36x36
            │   └── apps
            │       └── libresplit.png
            ├── 48x48
            │   └── apps
            │       └── libresplit.png
            ├── 512x512
            │   └── apps
            │       └── libresplit.png
            ├── 64x64
            │   └── apps
            │       └── libresplit.png
            ├── 72x72
            │   └── apps
            │       └── libresplit.png
            └── 96x96
                └── apps
                    └── libresplit.png

35 directories, 16 files
```

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**